### DISCUSSION TITLE
Ideas, Suggestions, etc

### DISCUSSION THREAD

**Discussion by ellisonleao** [ellisonleao/gruvbox.nvim#134] [Ideas]
Let's use this discussion topic for suggestions and ideas. Feel free to add your thoughts here.

**Comment by bakageddy** [comment]
Add a telescope theme for this beautiful theme?
![2022-08-04_23-08-1659635280](https://user-images.githubusercontent.com/91826760/182917341-373e633b-3ede-482a-b91f-12fc168c93fe.png)

Like this one...
